Shop_GXPEngine_base/GXPEngine/Shop/View/ShopMessageView.cs

```diff
@@ -5,54 +5,42 @@
 
 namespace View
 {
-    //This class will draw a messagebox containing messages from the Shop that is observed.
+    // This class will draw a messagebox containing messages from the Shop that is observed.
     public class ShopMessageView : Canvas
     {
         const int FontHeight = 20;
 
         private ShopModel shop;
 
-        //------------------------------------------------------------------------------------------------------------------------
-        //                                                  ShopMessageDisplay()
-        //------------------------------------------------------------------------------------------------------------------------
         public ShopMessageView(ShopModel shop) : base(800, 100)
         {
             this.shop = shop;
         }
 
-        //------------------------------------------------------------------------------------------------------------------------
-        //                                                  Step()
-        //------------------------------------------------------------------------------------------------------------------------
         public void Step()
         {
             DrawBackground();
             DrawMessages();
         }
 
-        //------------------------------------------------------------------------------------------------------------------------
-        //                                                  DrawBackground()
-        //------------------------------------------------------------------------------------------------------------------------
-        //Draw background color
         private void DrawBackground()
         {
+            // Draw background color
```

Comment:
This comment is inconsistent with the (new) overall commenting style of this file. The single-line comments (that have been leftover from the giant bulky method title comments) have usually still appeared above the actual method declarations as opposed to here. It's a small thing, but I wanted to point it out.